ui/scss/component/_wallet-tip-send.scss

```diff
@@ -2,3 +2,19 @@
   margin-top: -21px;
   margin-bottom: -21px;
 }
+
+.wallet-send-tip-modal {
+  .card__title {
+    margin-top: -16px;
```

Comment:
If you'd like @infinite-persistence I can take a look at adding these using `_vars` `--spacing` variables